### ISSUE TITLE
Joomla 5: warnings in update check

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1673]
Warning: Undefined property: stdClass::$failed in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 23
Warning:  Undefined property: stdClass::$current in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 28 | Warning:  Undefined property: stdClass::$current in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 39
Warning: Undefined property: stdClass::$versiondetail in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 55	
Warning: Undefined property: stdClass::$date in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 62
Warning: Undefined property: stdClass::$changes in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 70
Warning: foreach() argument must be of type array|object, null given in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 70
 Warning: Undefined property: stdClass::$installedversion in C:\xampp\htdocs\joomla\administrator\components\com_jem\views\updatecheck\tmpl\default.php on line 120

**Comment by mckillo** [comment]
@jojo12 This issue is becasuse the https://www.joomlaeventmanager.net/updatecheck/update_pkg_jem.xml file doesn't support Joomla 5.
This commit is the update XML file. This XML file must be update in https://www.joomlaeventmanager.net/updatecheck/update_pkg_jem.xml

**Comment by mckillo** [comment]
Please, tell me when the XML file was updated in the server. I want to test it too.

**Comment by ernst67** [comment]
@mckillo 
I just checked to update the file, but the live xml is quite different from the version here.

This is the live version:
https://www.joomlaeventmanager.net/updatecheck/update_pkg_jem.xml

I can make the amendmenst, but maybe it's smart to evatuate the differences first.. or update the version on Github

**Comment by mckillo** [comment]
@mckillo I agree, the update_pkg_jem.xml file is obsolete in the root of GitHub. That file is the previous version for JEM 2.3.5 and JEM 4.0.
I have updated the file with the XML file downloaded (live version) and updated to support J5 in update process.
Could you test it? Thanks,

**Comment by ernst67** [comment]
Excellent, thanks. 
The live file is now updated.

**Comment by conmarti** [comment]
commit https://github.com/jemproject/JEM-Project/commit/190105b02372753361ed07027306be0361b6e842 has made the xml file invalid.
notes should look like this
`<notes>JEM 4.1.0 for Joomla! 4.0, 4.1, 4.2, 4.3, 4.4 &amp; Joomla! 5.x</notes>`
otherwise you'll get an error in Joomla update check.
`Update: :Extension: Could not parse http://www.joomlaeventmanager.net/updatecheck/update_pkg_jem.xml`

**Comment by mckillo** [comment]
@ernst67 In the previous file, I wrote a '&' in the note string (line 81) and the XML parse (simplexml_load_string) doesn't allow. In the last commit is solved. You can upload the new update_pkg_jem.xml or change the '&' in the line 81 to 'and' in this file.

**Comment by mckillo** [comment]
When you upload the new file to the update server, the Check Update page in BE, it must show:

![image](https://github.com/jemproject/JEM-Project/assets/2765316/80caf9d4-8d7d-43ba-97ef-9552df5fbd71)

**Comment by jojo12** [comment]
now it works
